### DISCUSSION TITLE
git.exe launching in new window

### DISCUSSION THREAD

**Discussion by Devilsbane7** [git-for-windows/git#4152] [Q&A] [answered]
I'm upgrading git for windows from 2.27.0 to 2.38 and I'm noticing a strange behavior. Running any git command launches a new window. The git command runs over there nearly instantly and then the window disappears. I've done a video of my git config --list and can see the output gets displayed when I playback frame by frame, but since it's launching in a separate window I don't get a chance to read it before it disappears.

Any idea what's going on here? Could this have anything to do with using mintty? And if so, can I disable it so that the output of my git commands stays in my command prompt? (The same behavior happens if you run a git command from powershell or in git-cmd or git-bash)
![2022-12-08_16-18-21](https://user-images.githubusercontent.com/120140876/206579214-bb03f638-1668-4205-ad05-f5e3ee873762.png)
![2022-12-08_16-24-52](https://user-images.githubusercontent.com/120140876/206580561-81ea9243-81af-4ee6-bebe-21ff2690bd2f.png)
![2022-12-08_16-25-05](https://user-images.githubusercontent.com/120140876/206580576-bd14b7f0-d3c9-4cd5-8339-cf2290f074de.png)

I'm running Windows 10

**Comment by dscho** [comment]
Could have something to do with Pseudo Console support.

Maybe you could pinpoint the regression in the [snapshots](https://wingit.blob.core.windows.net/files/index.html), that might help us identify the cause quicker.

**Reply by Devilsbane7** [reply]
Thanks for the idea

**Comment by Devilsbane7** [comment, marked as answer]
It turns out, the new version was just a coincidence. I found a policy change implemented last week that someone had put in to elevate git to run as admin. Disabling that policy restored the previous behavior. Since the command prompt wasn't elevated, a new window needed to be launched to run it.
